### PR TITLE
set up a smartflix rails app with capybara test

### PR DESCRIPTION
**Pull request by chillkang**
- Set up a new Ruby on Rails app called smartflix.
- Write an acceptance test using Capybara that asserts: when the user visits the index route (/), the page content includes Smartflix.
- Create a new route, controller, and view to pass the acceptance test. Make sure you're following the Rails conventions.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -70,3 +70,4 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+gem "rspec-rails", group: [:development, :test]
```
Comment (the-harry) [inline Gemfile]: perhaps this could be moved to line 53?

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```
Comment (Swiftda01) [inline app/helpers/pages_helper.rb]: It's a good practice to avoid having empty boilerplate code

File: .rspec [hunk 3]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (the-harry) [inline .rspec:1]: This is just something I like to do, but it's nice to run the specs in a random order so you can spot easily flaky specs.

```suggestion
--require spec_helper
--order random
```

File: spec/features/homepage_spec.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+feature "homepage" do
+  scenario "enables me to view 'smartflix'" do
```
Comment (Swiftda01) [inline spec/features/homepage_spec.rb]: Typically a scenario would be a description of what the user is doing, so something like "A user visits the homepage"

File: Gemfile (hunk 1 above)
Comment (Swiftda01) [inline Gemfile]: Seems like you already have a development/test group, perhaps you could put this dependency in there to avoid duplication

**Review by Swiftda01 — APPROVED** [review]
A couple of small comments, otherwise looks great!

File: config/routes.rb [hunk 5]
```diff
@@ -3,4 +3,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get '/', to: 'pages#index'
```
Comment (the-harry) [inline config/routes.rb]: Perhaps would be better to add as a root_path:

```suggestion
  root to: 'pages#index'
```

**Review by the-harry — APPROVED** [review]

File: app/helpers/pages_helper.rb (hunk 2 above)
Comment (chillkang) [reply app/helpers/pages_helper.rb]: I deleted the empty boilerplate code!

File: spec/features/homepage_spec.rb (hunk 4 above)
Comment (chillkang) [reply spec/features/homepage_spec.rb]: Thank you for your suggestion! I changed the scenario accordingly.

File: Gemfile (hunk 1 above)
Comment (chillkang) [reply Gemfile]: I moved it to a development/test group.

Comment (chillkang) [reply Gemfile]: I moved it to a development/test group.

File: .rspec (hunk 3 above)
Comment (chillkang) [reply .rspec:1]: Thank you for your suggestion :)

File: config/routes.rb (hunk 5 above)
Comment (chillkang) [reply config/routes.rb]: I changed it accordingly.

**Review by the-harry — APPROVED** [review]

**Pull request merged**